upload/admin/view/template/extension/extension.tpl

```diff
@@ -19,38 +19,36 @@
         <fieldset>
           <legend><?php echo $text_type; ?></legend>
           <div class="well">
-            <div class="input-group">
-              <select name="type" class="form-control">
-                <?php foreach ($categories as $category) { ?>
-                <?php if ($type == $category['code']) { ?>
-                <option value="<?php echo $category['href']; ?>" selected="selected"><?php echo $category['text']; ?></option>
-                <?php } else { ?>
-                <option value="<?php echo $category['href']; ?>"><?php echo $category['text']; ?></option>
-                <?php } ?>
-                <?php } ?>
-              </select>
-              <span class="input-group-addon"><i class="fa fa-filter"></i> <?php echo $text_filter; ?></span>
+            <div class="btn-group ext-types">
+              <?php foreach ($categories as $category) { ?>
+              <?php $active = ($type == $category['code'])?' active':''; ?>
+              <button type="button" class="ext-type btn btn-default<?php echo $active; ?>" data-value="<?php echo $category['href']; ?>"><i class="fa fa-filter"></i> <?php echo $category['text']; ?></button>
```

Comment:
I would recommend removing the filter icon from each button:
`<i class="fa fa-filter"></i> `